pkg/kubectl/cmd/logs.go

```diff
@@ -199,7 +219,23 @@ func (o LogsOptions) Validate() error {
 
 // RunLogs retrieves a pod log
 func (o LogsOptions) RunLogs() (int64, error) {
-	req, err := o.LogsForObject(o.Object, o.Options)
+	switch t := o.Object.(type) {
+	case *api.PodList:
+		cnt := int64(0)
+		for _, p := range t.Items {
+			fmt.Fprintf(o.Out, "======== %s\n", p.Name)
```

Comment:
is this the separator for logs from multiple pods ? what about multiple containers in a pod